.docker.env.example

```diff
@@ -0,0 +1,4 @@
+DB_HOSTNAME=host.docker.internal
```

Comment:
This differs between platforms linux, mac and windows all do it differently, it's safer (from a ux perspective) to just set network_type: host and use localhost instead.

Reply:
Ah, it definitely works on Mac, and it works on Windows (based on reading the docs). I think this is the new recommended way to do it.

Reply:
https://stackoverflow.com/a/50057593